crates/agglayer/tests/snapshots/config__config_display.snap

```diff
@@ -35,6 +35,7 @@ settlement-timeout = 1200
 [l1]
 chain-id = 1337
 node-url = "http://zkevm-mock-l1-network:8545/"
+ws-node-url = "ws://zkevm-mock-l1-network:8546/"
```

Comment:
Would it make sense to only have one L1 node url, the protocol to be determined by the scheme (`http://` or `ws://`)? The `ws://` variant required for block based epoch clock.

Reply:
I would prefer to have different one as `http` and `ws` are sometimes on different ports